src/pages/HomePage/Framework/Framework.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { IoIosArrowDown, IoIosArrowUp } from 'react-icons/io';
+import { GoGitBranch } from 'react-icons/go';
+
+import type { Framework } from 'types/languages';
+
+import { LevelComponent } from '../Level/Level';
+
+import styles from './Framework.module.scss';
+
+interface FrameworkProps {
+  framework: Framework;
+}
+
+export const FrameworkComponent: React.FC<FrameworkProps> = ({ framework }) => {
+  const [showChildren, setShowChildren] = useState(false);
+  const changeShowChildrenState = () => {
+    setShowChildren(!showChildren);
```

Comment:
when new state is based on old state, functional form of setState is recommended, also consider reanming to: childrenVisible, setChildrenVisible, toggleChildrenVisibility (more intuitive to me, but ofc it's subjective)